.coderabbit.yaml

```
@@ -0,0 +1,42 @@
+  ignore:
+    - "node_modules/**"
+    - "dist/**"
+    - "build/**"
+    - "*.log"
+    - ".env*"
+    - "package-lock.json"
+
+chat:
+  auto_reply: true
+
+language: en-US
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)